architecture/index.adoc

```diff
@@ -0,0 +1,71 @@
+{product-title} assigns hosts different roles. These roles define the function of a machine in a cluster.
+The cluster contains definitions for the standard master and worker role types.
+
+You can use operators to package, deploy, and manage services on the control plane. With operators, you can use `kubectl` and `oc` commands.
+Operators are important components in {product-title} because they:
+
+* Perform health checks.
+* Enable the way to watch an application.
+* Manage over-the-air updates.
+* Ensure applications stay in the specified state.
+
+[id="how-to-develop-your-containerized-application"]
+== How to develop your containerized application
+
+As a developer, you can xref:../architecture/understanding-development.adoc#understanding-development[develop your containerized application] in many ways and there are different approaches based on requirements.
+You can use different tools, methods, and formats to develop your containerized application. You can build a simple container application with the various container build tool, base image, and registry options.
+You can use supporting components such as OperatorHub and templates to develop your application.
+If you want to make your application, you can package and deploy your application as an Operator.
+
+You can also create a Kubernetes manifest and store it in a Git repository.
+Kubernetes works on basic units called `pods`. Pods are the next step in application development.
+A service is a grouping of a logical set of pods and access policies. Services provide permanent internal IP addresses and host names for other applications to use as pods are created and destroyed.
+Based on the type of your application, Kubernetes defines a specific type of workload.
+
+[id="coreOS"]
+== All you need to know about Red Hat Enterprise Linux CoreOS (RHCOS)
+
+As a cluster administrator, you can
+
+** Know all about the key features of the next generation of xref:../architecture/architecture-rhcos.adoc#architecture-rhcos[single-purpose container operating system technology].
+** Choose how to configure RHOCS
+** Choose how to deploy RHOCS:
+*** Installer-provisioned deployment
+*** User-provisioned deployment
+
+[id="what-you-can-do-with-ignition"]
+== What you can do with Ignition
+The {product-title} installation program creates the ignition configuration files that you need to deploy your cluster.
+RHOCS uses Ignition for common disk tasks such as partitioning disks,formatting partitions, writing files, and configuring users, during the initial configuration.
+During the first boot, Ignition reads its configuration from the installation media or the location that you specify and applies the configuration to the machines.
+
+You can learn how Ignition works, the process for an RHCOS machine in an {product-title} cluster, view Ignition configuration files, and change Ignition configuration after an installation.
```

Comment:
This says you can learn these things, but it doesn't point to where. Is this describing what the Architecture documentation informs you about? If so, perhaps something like this would make sense?
```suggestion
The {op-system} Architecture (link: https://deploy-preview-40012--osdocs.netlify.app/openshift-enterprise/latest/architecture/architecture-rhcos.html) documentation can help you learn more about the following topics:
* How Ignition works
* The life cycle of an {op-system} machine in an {product-title} cluster
* Viewing Ignition configuration files
* Changing Ignition configuration files after an installation
```

Reply:
Can we use netlify preview link, I am not sure?

Reply:
Yes it describes the existing Arch doc

Reply:
I have included xref.... to the assembly that has the Ignition module.